### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ursjoss**
* Remove links to beehive and issuestats
* Update links to travis
* update versions of wicket-bootstrap modules in the `pom.xml` samples
* Update link to 'wicket-bootstrap in action'
* Update the version of bootstrap used in the wicket-8.x branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,19 @@ WICKET COMPONENTS FOR BOOTSTRAP
 
 [![Join the chat at https://gitter.im/l0rdn1kk0n/wicket-bootstrap](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/l0rdn1kk0n/wicket-bootstrap?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Wicket-Bootstrap is based on the Bootstrap toolkit and the [Apache Wicket Framework](https://wicket.apache.org/).
+Wicket-Bootstrap is based on the Bootstrap toolkit and the Apache Wicket Framework.<br>
```
Comment (martin-g) [inline README.md]: Why did you remove the link ?

**Comment by ursjoss** [comment]
I think I copied the Readme from another branch that didn't had the link and did not think of putting it back. I'll do so when back home...

File: README.md (hunk 1 above)
Comment (ursjoss) [reply README.md]: Fixed.
